### DISCUSSION TITLE
Tracking the Movement of a Mouse across Three Different Regions

### DISCUSSION THREAD

**Discussion by Dauniel** [talmolab/sleap#1685] [Help!]
Hi I am new to using SLEAP and was wondering if there is a functionality within SLEAP where I can track the movement of the mouse throughout a video and count how many times it went to region A, region B, and region C for example.

**Comment by Marielamend** [comment]
Hi @Dauniel!

First off welcome to SLEAP, hope it is going well! Here is a similar post to your inquiry and what we suggested!
> To tackle this, you need to train two separate models with different skeletons

(one consisting of rat body parts and the other representing the different regions) 
> and then run them in the same video. Make sure that each model with its respective skeleton is made in a separate project. This way you can get the coordinates you want for each instance.
#1645 

For post-processing you can check out [DeepOF](https://deepof.readthedocs.io/en/latest/) where you can extract trajectories and distances from your projects!

Let us know if you have any other questions, we are happy to assist!

Best, 
Mariela
